openapi/backend/openapi.yaml

```diff
@@ -351,8 +352,8 @@ paths:
         - $ref: '#/components/parameters/paginationTokenParam'
         - $ref: '#/components/parameters/paginationSizeParam'
         - in: query
-          name: featureIds
-          description: A comma-separated list of feature IDs to filter results.
+          name: featureId
+          description: A list of feature IDs to filter results.
```

Comment:
Maybe mention that this is supplied via repeated parameter names?

Reply:
Done. Thanks!